### ISSUE TITLE
ZeroTier DNS not working on Windows but working perfectly on Linux

### ISSUE THREAD

**Issue by wxrl** [zerotier/zeronsd#99]
So here is my setup

I have 2 machines: one is my personal Server running Debian 10 with nginx/php/mysql and samba, let's call it **deb-srv**, and the other is my Laptop, which is installed with Linux Mint 20.1 together with Windows 10 Home (dual systems, not on VM) and is under another physical sub network. 

I have ZeroTier and zeronsd installed and configured on **deb-srv** using **intra.mydomain.com**, and all 3 systems joined in the same ZeroTier network (so there are 3 nodes under the network, but always up to 2 nodes are online), for which I have verified by connecting via ZeroTier Intranet IPs.

I CAN connect to **deb-srv** using **intra.mydomain.com** (ping, ssh, http/https, samba) **from my Laptop when under Linux Mint**, but if I reboot to Windows on the same laptop (still under the same physical network), I can **only** connect to **deb-srv** using the ZeroTier Intranet IP but not **intra.mydomain.com**.

I have confirmed on either GUI and command line (powershell) that allowDNS is TRUE for the Windows client, and I have tried to restart the whole Windows system and the windows ZeroTier-One client for several times, but still not working. 

Also I tried to set manually again by zerotier-cli set network-id allowDNS=1, as well as un-check and re-check the allowDNS checkbox and restart the client - still no hope

I also tried under Administrator to **ipconfig /flushdns** as well as **Clear-DnsClientCache** and then reboot the system - still no hope...

I also tried to quit from the ZeroTier network from Windows and re-join, still nothing changed...

**Under Windows on the Laptop:**

> ping intra.mydomain.com
> Ping request could not find host intra.mydomain.com. Please check the name and try again.

> ping 172.30.*.*

> Pinging 172.30.*.* with 32 bytes of data:
> Reply from 172.30.*.*: bytes=32 time=27ms TTL=64
> Reply from 172.30.*.*: bytes=32 time=27ms TTL=64

**Comment by wxrl** [comment]
> 
> 
> Hello. Is the Windows VM on a domain? Check here https://discuss.zerotier.com/t/new-dns-features-dont-appear-to-work/982/5

The Windows is on the same physical PC with my Linux (Dual Systems)

**Comment by erikh** [comment]
so both machines are named intra? and each has their own, independent identity?

that's basically a collision. we don't support that at the moment, largely for this reason.

If you want to clone the identity between the windows and linux pcs so they are -- in the eyes of zerotier -- the same machine, that would almost certainly work.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by wxrl** [comment]
No.  The intra is only the name for my Debian Server (which is another physical machine).

I mean I have client_linux and client_windows on another dual boot machine. And from client_linux on this machine, I can visit intra.mydomain.com (to my Debian Server) without problem, but if I reboot to client_windows on this same machine it doesn't recognize intra.mydomain.com (Although I can access the Debian Server via the zerotier virtual IP)
